src/main/kotlin/app/jibiki/persistence/DatabaseAccessor.kt

```diff
@@ -119,19 +119,29 @@ FROM (SELECT json_build_object(
                      'id', source.id,
                      'language', source.lang,
                      'sentence', source.sentence,
+                     'audio_uri',
+                     (SELECT 'https://audio.tatoeba.org/sentences/' || source.lang || '/' || source.id || '.mp3'
```

Comment:
why are you so addicted to subselects? why not a simple case statement?

Reply:
oh yea I forgot those existed, thanks for reminding me